functions.c

```diff
@@ -323,7 +323,8 @@ const struct MenuFuncOp OpGeneric[] = { /* map: generic */
   { "previous-entry",                OP_PREV_ENTRY },
   { "previous-line",                 OP_PREV_LINE },
   { "previous-page",                 OP_PREV_PAGE },
-  { "refresh",                       OP_REDRAW },
+  { "redraw-screen",                 OP_REDRAW },
+  { "refresh",                       OP_REFRESH_DEPRECATED },
```

Comment:
If you move the original, "refresh" to the deprecated section below, that should be enough.